components/input_environmental_personal.py

```diff
@@ -1,26 +1,31 @@
 import dash_mantine_components as dmc
-from dash import html
-
-from utils.my_config_file import ModelInputsInfo, Models, convert_units, ElementsIDs
+from dash import html, Input, Output, State, callback
+from utils.my_config_file import ModelInputsInfo, Models, convert_units, ElementsIDs, MetabolicRateSelection,ClothingSelection
 
 
 def input_environmental_personal(selected_model: str = "PMV_ashrae", units: str = "SI"):
     inputs = []
     model = Models[selected_model].value.inputs
     convert_units(model, units)
 
-    values: ModelInputsInfo
     for values in model:
-        input_filed = dmc.NumberInput(
-            label=values.name + " (" + values.unit + ")",
-            description=f"From {values.min} to {values.max}",
-            value=values.value,
-            min=values.min,
-            max=values.max,
-            step=values.step,
-            id=values.id,
-        )
-        inputs.append(input_filed)
+        if values.name == "Metabolic Rate":
```

Comment:
we should never compare a string but compare it to the value in the class

Reply:
I'll deal with that